VMEncryption/main/handle.py

```diff
@@ -751,7 +751,11 @@ def enable():
         encryption_config = EncryptionConfig(encryption_environment=encryption_environment, logger=logger)
         if encryption_config.config_file_exists():
             existing_volume_type = encryption_config.get_volume_type()
-
+        #log to capture lsblk before encryption view.
+        disk_util.log_lsblk_output()
+        if public_settings.get(CommonVariables.EncryptionEncryptionOperationKey) == CommonVariables.EnableEncryptionFormatAll:
```

Comment:
Is the incident reported for EncryptFormalAll scenarios only?

Reply:
yes, in standard VM resource disk encryption is supported only for  EncryptFormalAll scenario. issue has been seen on stop start.

Reply:
Is the resource disk still getting encrypted if it is unmounted?

Reply:
yes, resource disk got encrypted if EFA is there, irrespective of disk mount. 
I had explained the issue in the below page. 
https://microsoft.sharepoint.com/teams/AzureSecurityCompliance/Security/_layouts/15/Doc.aspx?sourcedoc={c820c6d1-ba1e-45d8-bd74-5e103f5ac176}&action=edit&wd=target%28LiveSite.one%7Ccd0e0f5b-0d53-4726-81f4-85b86d4cfa01%2FAnalysis%20384350526%20Investigate%20latency%20Issues%20with%20Compute%20Instance%20LROs%7Cd38eeb58-1889-41cb-8e92-21bbb8cef4ff%2F%29&wdorigin=703

Reply:
Perhaps, we should consider mounting BEK volume on /bekvolume

Reply:
@canfikret yeah, we have been wanting to move it. For new VM's, it is an easy change, but for existing VM's, snapshots/backups/etc we would need to think about a solution that wont break those.